numpy/core/src/umath/_umath_tests.c.src

```diff
@@ -294,6 +295,91 @@ static void
 
 /**end repeat**/
 
+char *sumsquarediff_signature = "(i),(1)->(),()";
+
+static void
+HALF_sumsquarediff(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUSED(func))
```

Comment:
If these are used by any functions outside `test_*.py` (such as `np.var`), then they should not be in `_umath_tests`. You'll need to find somewhere else to put them